src/Uno.UI/Controls/AppBarButtonRenderer.iOS.cs

```diff
@@ -74,86 +73,93 @@ protected override IEnumerable<IDisposable> Initialize()
 					default:
 						this.Log().WarnIfEnabled(() => $"{GetType().Name ?? "FontIcon, PathIcon and SymbolIcon"} are not supported. Use BitmapIcon instead with UriSource.");
-						Native.Image = null;
-						Native.CustomView = null;
+						native.Image = null;
+						native.CustomView = null;
 						// iOS doesn't add the UIBarButtonItem to the native logical tree unless it has an Image or Title set. 
 						// We default to an empty string to ensure it is added.
-						Native.Title = string.Empty;
+						native.Title = string.Empty;
 						break;
 				}
 			}
 			else
 			{
-				switch (Element.Content)
+				switch (element.Content)
 				{
 					case string text:
-						Native.Image = null;
-						Native.CustomView = null;
-						Native.Title = text;
+						native.Image = null;
+						native.CustomView = null;
+						native.Title = text;
 						break;
 
 					case FrameworkElement fe:
-						var currentParent = Element.GetParent();
-						_appBarButtonWrapper.Child = Element;
+						var currentParent = element.GetParent();
+						_appBarButtonWrapper.Child = element;
 
 						//Restore the original parent if any, as we
 						// want the DataContext to flow properly from the
 						// CommandBar.
-						Element.SetParent(currentParent);
+						element.SetParent(currentParent);
 
-						Native.Image = null;
-						Native.CustomView = fe.Visibility == Visibility.Visible ? _appBarButtonWrapper : null;
+						native.Image = null;
+						native.CustomView = fe.Visibility == Visibility.Visible ? _appBarButtonWrapper : null;
 						// iOS doesn't add the UIBarButtonItem to the native logical tree unless it has an Image or Title set.
 						// We default to an empty string to ensure it is added, in order to support late-bound Content.
-						Native.Title = string.Empty;
+						native.Title = string.Empty;
 						break;
 
 					default:
-						Native.Image = null;
-						Native.CustomView = null;
+						native.Image = null;
+						native.CustomView = null;
 						// iOS doesn't add the UIBarButtonItem to the native logical tree unless it has an Image or Title set. 
 						// We default to an empty string to ensure it is added.
-						Native.Title = string.Empty;
+						native.Title = string.Empty;
 						break;
 				}
 			}
 
 			// Label
-			Native.AccessibilityHint = Element.Label;
-			Native.AccessibilityLabel = Element.Label;
+			native.AccessibilityHint = element.Label;
+			native.AccessibilityLabel = element.Label;
 
 			// Foreground
-			if (Brush.TryGetColorWithOpacity(Element.Foreground, out var foreground))
+			if (Brush.TryGetColorWithOpacity(element.Foreground, out var foreground))
 			{
 				var color = (UIColor)foreground;
-				Native.TintColor = color.ColorWithAlpha((nfloat)Element.Opacity);
+				native.TintColor = color.ColorWithAlpha((nfloat)element.Opacity);
 			}
 			else
 			{
-				Native.TintColor = default(UIColor); // TODO .Clear;
+				native.TintColor = default(UIColor); // TODO .Clear;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6270650)